.gitignore

```diff
@@ -1,6 +1,3 @@
-# Ignore composer.lock to allow highest/lowest testing.
```

Comment:
I think this comment may make sense, but we could just remove the file during tests and run tests after a composer update as well.

Reply:
Agreed, we'll do that later if it's necessary.